test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Get_ApplicationJson_DecimalType_HttpTrigger.cs

```diff
@@ -0,0 +1,31 @@
+using System.Net;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Models;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp
+{
+    public class Get_ApplicationJson_DecimalType_HttpTrigger
```

Comment:
Can you change both file name and class name to `Post_ApplicationJson_DecimalType_HttpTrigger`?